src/main/java/net/sf/jabref/importer/OpenDatabaseAction.java

```diff
@@ -454,4 +454,64 @@ private static InputStreamReader openFile(File fileToOpen, Optional<Charset> enc
         }
         return Optional.empty();
     }
+
+    /**
+    *
+    * @param name Name of the bib-file to open
+    * @param ignoreAutosave true if autosave version of the file should be ignored
+    * @return ParserResult which never is null
+    */
+
+    public static ParserResult openBibFile(String name, boolean ignoreAutosave) {
```

Comment:
Better name for sure. Any suggestion? Javadoc: I see the point, but not really sure what to write than, more or less, combine the argument descriptions "Open bib-file and possibly ignore autosave file", sort of.